preparation.py

```diff
@@ -45,7 +49,7 @@ def overlay_masks_from_annotations(annotations, image_size):
         m = cocomask.decode(rle)
         m = m.reshape(image_size)
         mask += m
-    return (255 * (mask > 0)).astype('uint8')
+    return np.where(mask > 0, 1, 0).astype('uint8')
```

Comment:
@apyskir this will return a mask of 1s and 0s wheras before it would return a mask of 255s and 0s. Was that intentional or typo?

Reply:
@jakubczakon It was intentional. Have a look at line 36 in preparation.py. It will exceed 255 if I return mask of 255s and 0s.

Reply:
@apyskir ok cool